tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/assign/LValueTest.java

```diff
@@ -83,9 +83,19 @@ public void testNegativeCases() {
         validateError(negativeResult, i++, "undefined field 'y' in record 'E'", 75, 5);
         validateError(negativeResult, i++, "invalid operation: type 'map<int>?' does not support member access for " +
                 "assignment", 78, 5);
-        validateError(negativeResult, i, "invalid operation: type 'E?' does not support field access", 79, 5);
+        validateError(negativeResult, i++, "invalid operation: type 'E?' does not support field access", 79, 5);
     }
 
+    @Test
+    public void testNegativeLvexpr() {
+        CompileResult negative = BCompileUtil.compile("test-src/statements/assign/lvexpr_negative.bal");
+        int i = 0;
+        validateError(negative, i++, "invocations are not supported in lhs of an assignment", 26, 5);
```

Comment:
shouldn't this be updated to reflect the change to the error message?

Reply:
Yes I forgot